src/assets/languages/languageSelector.js

```diff
@@ -110,7 +110,7 @@ const languageSelector = (lang, str) => {
     restaurantOptionsReview: "REVIEWS",
     overViewSchedule: "Opening Hours",
     overviewFacilities: "Facilities",
-    bookingButton: "SUBMIT",
+    bookingButton: "SUMBIT",
```

Comment:
this was corrected in another PR, why is this typo here?

Reply:
I don't know what happened, still resolved.